### PR TITLE
Deployment introduction /  index page

### PR DESCRIPTION
**Pull request by LaraAS**
Adding the index file that introduces the different steps entailed in deploying an SNS and points to each of the more detailed pages.
Covers this Jira ticket: https://dfinity.atlassian.net/browse/NNS1-1621

### CODE REVIEW

File: docs/developer-docs/functionality/sns/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# How to launch an SNS
```
Comment (sesi200) [inline docs/developer-docs/functionality/sns/index.md:1]: Should this page start with what an SNS is?

File: docs/developer-docs/functionality/sns/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+We explain how to deploy an SNS that is provided as a 
+system functionality by the IC.
+As explained in the [introduction](sns.md), there are also other
```
Comment (sesi200) [inline docs/developer-docs/functionality/sns/index.md:5]: Does this mean there are two introductory/index-like pages that should be merged? ('No' is a complete answer, too)

File: docs/developer-docs/functionality/sns/index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+We introduce here the steps to go through to launch an SNS.
+We only give an overview here and, for each of the steps, we 
+link to a page that provides more details. 
+
+### Step 1: Defining the goals (non-technical preparation)
+The successful launch of an SNS does not only depend on technical
+aspects. 
+In fact, it is advisable that before taking any technical steps, 
+you take some non-technical preparations steps and define what the SNS
```
Comment (sesi200) [inline docs/developer-docs/functionality/sns/index.md:17]:
```suggestion
you take some non-technical preparation steps and define what the SNS
```

**Review by sesi200 — APPROVED** [review]

File: docs/developer-docs/functionality/sns/index.md (hunk 1 above)
Comment (LaraAS) [reply docs/developer-docs/functionality/sns/index.md:1]: There will be another page explaining what an SNS is /functionality/sns and this here should only be the introduction to how to deploy an SNS.

File: docs/developer-docs/functionality/sns/index.md (hunk 2 above)
Comment (LaraAS) [reply docs/developer-docs/functionality/sns/index.md:5]: No :) One page introduced the SNS (the one that is linked) and this page here is actually an index to "how to launch an SNS" that then has multiple sub-pages.. 
If there is a better way to reflect that in the structure, please feel free to suggest something!

Comment (LaraAS) [reply docs/developer-docs/functionality/sns/index.md]: renamed the file and will resolve this thread

**Pull request merged**